### DISCUSSION TITLE
No paste if document is close

### DISCUSSION THREAD

**Discussion by ghost** [hluk/CopyQ#1983] [General]
Sorry, but I don't understand ....
I open a text file,
I copy a word and I close the document
I can't use paste because nothing happens ......
I must open CopyQ and double click to paste the word ....
So, if I close the document I lose the word, is ok if I don't close the document ....
How can I fix this issue?

SO Kubuntu 20.04 - CopyQ 6.1.0

**Comment by hluk** [comment]
Usually, if you close the application that owns the clipboard (where you copied a content the last time) it won't be able to provide the content anymore (and the clipboard is cleared).

As workaround, CopyQ can **revert the last clipboard content** when it is cleared. To enable this, **add [Reset Empty Clipboard/Selection](https://github.com/hluk/copyq-commands/tree/master/Scripts#reset-empty-clipboardselection) command** (see [README.md](https://github.com/hluk/copyq-commands/blob/master/README.md) for details on how to add the command).

**Reply by ghost** [reply]
Ok, thank you ....
